ui/src/components/directory/viewer/set/SetDefinition.vue

```diff
@@ -247,10 +247,12 @@ async function download(): Promise<void> {
   try {
     toast.add(new ToastOptions(ToastSeverity.SUCCESS, "Download will begin shortly"));
     const schemes = [] as string[];
-    if(selectedSchemes.value.length !== 0) {
+    if (selectedSchemes.value.length !== 0) {
       selectedSchemes.value.forEach(s => schemes.push(s["@id"]));
     }
-    const result = (await EntityService.getFullExportSet(props.entityIri, definition, core, legacy, checked.value, checkedLegacy.value, im1id, selectedFormat.value,schemes)).data;
+    const result = (
+      await EntityService.getFullExportSet(props.entityIri, definition, core, legacy, checked.value, checkedLegacy.value, im1id, selectedFormat.value, schemes)
```

Comment:
I'm going to pick up this. Try/catch is too general. Needs specific error scenario handlers.